tests/jerry/es.next/object-get-own-property-descriptors.js

```diff
@@ -0,0 +1,107 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var object = {};
+
+var descriptor1 = {
+  value: 42,
+  writable: true,
+  enumerable: true,
+  configurable: true,
+};
+
+var descriptor2 = {
+  value: "foo",
+  writable: true,
+  enumerable: false,
+  configurable: true,
+};
+
+var descriptor3 = {
+  value: undefined,
+  writable: false,
+  enumerable: true,
+  configurable: false,
+};
+
+Object.defineProperties(object, {
+  "property1": descriptor1,
+  "property2": descriptor2,
+  "property3": descriptor3,
+});
+
+var descriptors = Object.getOwnPropertyDescriptors(object);
+
+assert(JSON.stringify(Object.getOwnPropertyNames(descriptors)) === '["property1","property2","property3"]');
```

Comment:
Please avoid comparing arrays as JSON strings. There should be an `array_check` method in other test cases which can be used here also.